src/state-summary/state-card-climate.html

```diff
@@ -52,14 +56,23 @@
       type: Boolean,
       value: false,
     },
-
     stateObj: {
       type: Object,
     },
   },
 
   computeTargetTemperature: function (stateObj) {
-    return stateObj.attributes.temperature + ' ' + stateObj.attributes.unit_of_measurement;
+    if (stateObj.attributes.operation_mode.includes('Auto')) {
```

Comment:
There will be some variations on how this string is, ie auto, auto changeover, Auto, Auto Changover... Is there a regex helper that can be used?